docs/transaction-building.md

```diff
@@ -0,0 +1,264 @@
+  { sum: false } // [!code focus]
+);
+```
+
+:::
+
+### Step. 4.2: Mint a token
+
+Tokens on Ergo can be minted, that is, created, by a transaction. To create a new token, set the `Token ID` equal to the first input's `Box ID` and add it to the outputs. Additionally, [EIP-4](https://github.com/ergoplatform/eips/blob/master/eip-0004.md) defines a pattern for uniform token minting across the ecosystem by settings values on additional registers.
+
+The `OutputBuilder`'s `mintToken()` method provides a seamless way to mint EIP-4 tokens.
+
+In the following example, we are minting a token named `TestToken` and sending it to `9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j` address.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  .from(inputs)
+  .to(
+    new OutputBuilder("1000000", "9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j")
+      .mintToken({ // [!code focus]
+        amount: "100", // the amount of tokens being minted without decimals // [!code focus]
+        name: "TestToken", // the name of the token // [!code focus]
+        decimals: 2, // the number of decimals  // [!code focus]
+        description: "This is a test token minted with Fleet SDK" // [!code focus]
+      }) // [!code focus]
+  );
+```
+
+:::warning
+Only one token can be minted per transaction.
+:::
+
+## Step. 5: Set the change address
+
+Often, all the funds included in the input boxes are not used by the transaction. An address needs to be set to receive these funds. This is referred to as the change from the eUTxO transaction.
+
+To do so, you must call the `sendChangeTo()` method and pass an address as the only parameter.
+
+In the following example, we are sending the change to the `9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j` address.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .sendChangeTo("9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j"); // [!code focus]
+```
+
+## Step. 6: Set the fee amount
+
+Each transaction on Ergo needs to have a mining fee box, you can use the `payFee()` method to specify the fee amount in `nanoergs`.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .payFee("1100000"); // [!code focus]
+```
+
+Alternatively, you can use the `payMinFee()` method to add the min recommended miner fee amount, which currently is `1100000 nanoergs` or `0.0011 ERG`.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .payMinFee(); // [!code focus]
+```
+
+## Step. 7: Build
+
+Now that all pieces are put together, it's time to build the transaction and obtain the **Unsigned Transaction** object. For that you can use the `build()` method.
+
+The `build()` method must be the last called method on the `TransactionBuilder` as it will do all necessary validations, input selection, change calculations and return an unsigned transaction object which can be signed by any Ergo signing tool.
```

Comment:
```suggestion
The `build()` method must be the last called method on the `TransactionBuilder` as it will do all necessary validations, input selection, change calculations and return an unsigned transaction object which can be signed by any Ergo signing tool, such as a wallet.
```

Reply:
Signing tool is inclusive, but it is also vague. I wanted to added a concrete, and typical, example of signing tool. Also, good bridge to next part, which is also one option among many, such as Ergopay signing.